src/ec/packages/ec-component-text-input/docs/text-input.md

```diff
@@ -0,0 +1,45 @@
+# ECL Twig - EC Text Input component
+
+npm package: `@ecl-twig/ec-component-text-input`
+
+```shell
+npm install --save @ecl-twig/ec-component-text-input
+```
+
+## Text input
+
+### Parameters:
+
+- "id" (string) (default: '')
+- "disabled" (boolean) (default: false)
+- "invalid" (boolean) (default: false)
+- "invalidIconLabel" (string) (default: '')
```

Comment:
We need to talk about the naming conventions: either we stick to snake_case or to camelCase, but we don't mix both

Reply:
Actually, everywhere else, snake_case is used :) no need to discuss it, let's just update these variables names